src/Uno.Extensions.Core.Generators/Uno.Extensions.Core.Generators.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<IsPackable>false</IsPackable>
+		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
```

Comment:
Are RuntimeIdentifiers required?

Reply:
Well you add them in reactive package https://github.com/unoplatform/uno.extensions/pull/348 but I don't known why. I kept the same config to reduce risk of regression, but I agree that I don't see why there are needed. Why did you add them by that time @nickrandolph ?

Reply:
Unless I'm mistaken I think we've removed the RuntimeIdentifiers from both Core and Reactive projects - they were needed at some point for things to build but no longer an issue afaik for netstandard projects

Reply:
Ok then we should remove them from all projects